app/jobs/risc_delivery_job.rb

```diff
@@ -8,15 +8,21 @@ class RiscDeliveryJob < ApplicationJob
     Errno::ECONNREFUSED,
   ].freeze
 
+
   retry_on(
     *NETWORK_ERRORS,
     wait: :exponentially_longer,
     attempts: 2,
   )
+
   retry_on RedisRateLimiter::LimitError,
            wait: :exponentially_longer,
            attempts: 10
 
+  def self.skip_log_on_failure?
+    true
```

Comment:
I think we still want to log failures for this job, but avoid the error log if we're on the last retry and the error is one in `NETWORK_ERRORS`?